site/en/docs/webstore/cws-enterprise/index.md

```diff
@@ -0,0 +1,143 @@
+
+Select this option to publish privately to your domain.
+
+For further instructions on how to publish to your organization's domain, see [Chrome Insider:
+Publishing custom extensions for the enterprise][ent-ext-blog-how].
+
+## Managing domain Chrome Web Store extensions {: #manage-domain-extensions }
+
+There are also features that you don't use directly as an extension developer, but which you should
+be aware of.
+
+**Collections:** Administrators in your organization can curate *collections* of extensions that are
+appropriate to your org. These collections can include both public and privately published
+extensions. These collections are like the ones in the public Chrome Web Store, such as Editor's
+Picks, but appear in your private Chrome Web Store.
+
+The article [Create a Chrome app collection][ent-ext-admin-collections] provides instructions for
+administrators to add collections to your private Chrome Web Store.
+
+**Blocklist and allowlist:** These are options that your organization's administrators can
+use to explicitly control which extensions may or that may not be installed by members of the org.
+
+These and other enterprise extension controls are set by administrators using policies in the
+[extensions atomic policy group][ent-ext-admin-policies].
+
+**Version pinning:** In an enterprise context, it can be a problem when software versions update
+unexpectedly; these changes can necessitate new training and procedures or interfere with
+integrations. For organizations with a private Chrome Web Store enabled, administrators can
+designate specific versions of extensions that users in the organization can install.
+
+## Non-webstore installations {: #side-loading }
+
+There are ways to install enterprise extensions without using the Chrome Web Store, such as those
+described in this section.
+
+**Force install:** Administrators can set policies that automatically install extensions
+for users in the organization. This is typically used for extensions that support workflows and
+other core business operation.
+  
+See the [ExtensionInstallForcelist][ent-ext-admin-forcelist] documentation to learn more about
+this feature.
+
+## Troubleshooting tips for domain extensions {: #troubleshooting }
+
+This section addresses a few common issues that extension developers may encounter when publishing
+to a domain.
+
+**I don't see the "Only to my domain" option under the private publishing option.**
+
+This option is only available to domain users when it has been enabled for the domain. Make sure that:
+
+* You are logged in using your domain identity, and not a public (gmail) or other non-domain identity.
+* Your administrator has configured domain publishing.
+
+**Why are my users not getting extension updates after I publish them?**
+
+It can take a while for Chrome to pull updates from the Chrome Web Store. Make sure your users are
+staying logged into Chrome with their domain identity for at least a few hours at a time.
```

Comment:
Another potential cause of issue here is that policies can take a while to propagate. If they just updated a policy to add a new extension, they may need to wait for it to actually be picked up by Chrome. [This support article](https://support.google.com/chrome/a/answer/187202?hl=en#zippy=%2Cwindows) touches a little on propagation at the end of the "Windows" section. I'm not sure how deep we want to get into the troubleshooting, but users can see Chrome's current policies by visiting `chrome://policy`.

Additional lag can also be introduced by the CWS's rollout process. This is probably best covered in a new page dedicated to extension updates, but the short version is that Omaha (the service that backs the Chrome and CWS update process) has differential serving and rolls out updates to an extension's user base over time.

Reply:
This is getting fairly involved, but it's a good point. Tracking as b/180055398.